#!/usr/bin/pyhon3
def raise_exception_msg(message=""):
    print("{}".format(message), end="")
    raise NameError
